warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/ShardedTableMapFile.java

```diff
@@ -349,7 +350,9 @@ public static Path createShardedMapFile(Logger log, Configuration conf, Path wor
                     Range range = new Range();
                     Locations locations = tableOps.locate(shardedTableName, Collections.singletonList(range));
                     List<TabletId> tabletIds = locations.groupByRange().get(range);
-                    tabletIds.forEach(tId -> splitToLocation.put(tId.getEndRow(), locations.getTabletLocation(tId)));
+                    
+                    tabletIds.stream().filter(tId -> tId.getEndRow() != null)
+                                    .forEach(tId -> splitToLocation.put(tId.getEndRow(), locations.getTabletLocation(tId)));
```

Comment:
Have you run a build with this file? The formatter that runs as a part of the build will likely re-format this line.